### PR TITLE
optionally get parameters from environment

### PR DESCRIPTION
**Pull request by andrewh1978**

### CODE REVIEW

File: start-slaves.sh [hunk 1]
```diff
@@ -36,14 +40,12 @@ function start_slave {
...
 
-export COMPOSE_PROJECT_NAME=$1
-export MESOS_MASTER=$2
 datacenters=( "us-east-3b" "us-east-1" "us-sw-1" "eu-ams-1" )
 for i in "${datacenters[@]}"
 do
 
     # don't create additional hosts for the current data center
-    if [ "tcp://$i.docker.joyent.com:2376" == "$3" ]
+    if [ "tcp://$i.docker.joyent.com:2376" == "$MARATHON" ]
```
Comment (misterbisson) [inline start-slaves.sh]: `$MARATHON` for the Docker host data center name?

File: start-slaves.sh [hunk 2]
```diff
@@ -8,9 +8,13 @@
...
 
-echo "     project prefix: $1"
-echo "       mesos master: $2"
-echo "current Docker host: $3"
+COMPOSE_PROJECT_NAME=${1:-${COMPOSE_PROJECT_NAME}}
+CONSUL=${2:-${CONSUL}}
+MARATHON=${3:-${MARATHON}}
+
+echo "     project prefix: $COMPOSE_PROJECT_NAME"
+echo "       mesos master: $CONSUL"
```
Comment (misterbisson) [inline start-slaves.sh]: `$CONSUL` for the Mesos master name?

**Comment by misterbisson** [comment]
@andrewh1978 sorry I didn't see this right away. I like your changes, but had some questions about the variable names. Do you want to revise?

**Comment by andrewh1978** [comment]
Embarrassing. Fixed variable names.

**Comment by misterbisson** [comment]
Thanks for fixing my spelling error along the way ;-)

**Pull request merged**